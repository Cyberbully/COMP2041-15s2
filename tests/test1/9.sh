#!/bin/sh

# Enter shell script or pipeline below.

for arg in "$@"
do
    sed -r -e 's/^([^aeiou]*)([aeiou])([^aeiou]*)([aeiou])/\1\4\3\2/gi' "$arg"
done
