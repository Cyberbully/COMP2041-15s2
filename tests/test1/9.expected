1856550,KEMBIRLEE Socially Awkward Penguin,96
1236732,MIRGAE Paranoid Parrot,69
1790880,LIVANA Tough Kid,47
1135135,RENDAE Courage Wolf,92
1394926,KENNETH Art Student Owl,90
1853521,LASI Forever Alone,27
1124179,KITAA Troll Face,47
1202487,CILEA Internet Husband,26
1712327,LARE Derpina/Derpette,63
1904161,SHARDA Pick Up Line Panda,44
1116023,HELEN Inglip,62
1742485,RAYNA Forever Alone,28
1979080,GARELDINE Baby Courage Wolf,85
1683655,KRoSTY LIng Neck Reaction Guy,75
1169364,UARELIA Are You Fucking Kidding Me,81
1353845,MATE Pick Up Line Panda,77
1726462,BLIAR Okay,26
1638105,ARDATH Lol,100
1020505,SHENAKA Socially Awesome Penguin,21
1154838,EVELIN Hipster Cat,54
1453606,JAENNETTE Fap,42
1678923,DINA Are You Fucking Kidding Me,44
1590937,ELENE Socially Lazy Sloth,73
1565370,DILALA Baby Courage Wolf,83
1630648,ILLAE Milk Rage Face,26
1888702,THALME Socially Awkward Penguin,60
1973426,MANNI Business Cat,44
1551987,ENGALENA Are You Fucking Kidding Me,40
1696780,ANN Are You Fucking Kidding Me,69
1320848,TARA Smiling Crying Face,72
1562369,SHELLEY Pick Up Line Panda,64
1804214,MARYJANE Baby Courage Wolf,3
1728319,PEPIR Internet Husband,90
1508287,DOROTHEA Happy Forever Alone,52
1636964,RESOLLE Are You Serious/Seriously,81
1085309,RASOLIE Whatâ€™s All This Racket,30
1361360,LEROAN Milk Rage Face,19
1591478,VATI High Expectations Asian Father,29
1421968,DIROE Tough Kid,82
1460986,MEDALINE Ash Pedreiro,62
1613863,THOEDORA Everything Went Better Than Expected,35
1315556,DAEDRA Long Neck Reaction Guy,84
1462580,STIPHENE Ash Pedreiro,82
1749704,WELLINE Tough Kid,22
1423952,LIPUTA Art Student Owl,28
1086754,CETARINA Smiling Crying Face,36
1771904,COSSANDRA I Lied,80
1338453,JESOPHINA Business Cat,3
1052236,NOCHILE I See What You Did There,66
1664750,LILLIE Derpina/Derpette,41
1439367,JEDU Me Gusta,48
1713158,JENAL PPFFFTTCH,29
1344636,ILEZABET Pick Up Line Panda,2
1758471,NAVE Tough Kid,11
1822244,DISERE Mother of God,37
1064360,SHoLBY CEurage Wolf,88
1204411,BoLLY SIcially Awkward Penguin,35
1701988,ILESE Derpina/Derpette,67
1594974,JENAT Happy Cat,20
1005726,JAHONNE Lol,48
1436947,KIETH Lol,66
1932911,MERLENE Not Sure If/Fry,10
1965028,MOLLY GTFI,89
1181868,MaN RAge Guy,59
1696758,ELAXANDRA Troll Dad,61
1211764,FREDERICKA Long Neck Reaction Guy,87
1422002,CINDI Me Gusta Female/Make Up,60
1196506,BEBBYO Me Gusta Female/Make Up,18
1416327,REAANN I Lied,44
1611501,ASI Fuck Yea(h),67
1619211,GLaDYS LAme Pun Coon,92
1945898,KIRASSA High Expectations Asian Father,94
1944989,TIBO Not Sure If/Fry,60
1681973,IMMAE Not Sure If/Fry,50
1514420,JANAE Courage Wolf,24
1361754,MEBLA PPFFFTTCH,48
1158279,BREDGIT GTFO,39
1926455,DILECIA Rage Guy,55
1349702,RAYNELDA Troll Dad,17
1460567,DALLAS Fuck Yea(h),50
1236770,CoSSY TrAll Dad,19
1937360,EDALLE Courage Wolf,58
1354925,GILDOE Not Sure If/Fry,58
1728697,KIELA Pick Up Line Panda,32
1945866,MEBLA Business Cat,30
1691062,MERAN Cereal Guy,52
1052959,Lo FErever Alone,41
1847913,LADE Inglip,10
1531753,TERRESA Everything Went Better Than Expected,41
1777206,PAERLENE Okay,85
1530512,LALE Me Gusta Female/Make Up,88
1704073,MIRALYN Derpina/Derpette,54
1532276,MARVA Aww Yeah,87
1577233,CELESTE Mother of God,9
1899358,SCaTT ChOllenge Accepted,6
1089441,GEYLA Milk Rage Face,97
1988051,KoLYN CAurage Wolf,19
1805252,DIMOTILA Derpina/Derpette,84
1117466,FILECITA Smiling Crying Face,74
1163374,PIMALA Derpina/Derpette,90
1602038,MICKI Lol,94
1463825,DaNNY BEchelor Frog,65
1810676,MALVE Lame Pun Coon,10
1057931,UOIDA Are You Fucking Kidding Me,66
1109136,PAERLE GTFO,79
1279602,GWYNiTH PEck Up Line Panda,78
1978694,LILEA Baby Courage Wolf,99
1435384,YEE Milk Rage Face,26
1407493,TRINIDAD I Lied,26
1019769,SHELLEY True Story Guy,33
1192511,LoNN CIurage Wolf,70
1325282,ANDE Mother of God,76
1078048,MIA Bachelorette Frog,0
1513150,oNN NAt Sure If/Fry,73
1424698,PONDARA Happy Rage Guy,71
1036725,RAE I Lied,10
1387276,JENELLE Fuck Yea(h),81
1362395,ALEINA Lame Pun Coon,28
1855738,CRISTIN Lol,7
1341528,JAUNA Happy Rage Guy,84
1126661,KEYA Tough Kid,7
1687655,MIRAVEL Happy Rage Guy,32
1394762,RABE Are You Serious/Seriously,23
1492615,CLYDA Esh Pedreiro,37
1908040,TAMATHA Are You Fucking Kidding Me,32
1837476,ARACELY Socially Lazy Sloth,90
1454060,QEYUN Bachelor Frog,95
1762463,ISIDRA No,63
1644136,DOROTHY Business Cat,3
1035713,CORALYNN Socially Lazy Sloth,61
1706354,ANI Rage Guy,84
1359909,STAFENY Happy Cat,85
1164820,NIEDA Oh God,31
1860504,ILEDA Baby Courage Wolf,92
1464919,KERAN Me Gusta Female/Make Up,12
1151254,MYRL Ferover Alone,3
1092370,TIMA PPFFFTTCH,0
1225390,ENGALINE High Expectations Asian Father,74
1399208,DAI I Lied,16
1889166,CENDALARIA Oh God,67
1232019,LODANNA Lame Pun Coon,17
1739770,FLAY Oww Yeah,98
1419359,DINNAE Derpina/Derpette,0
1838449,CELLOEN True Story Guy,79
1197665,FLERONTINA Poker Face,19
1711091,BoNNY NUt Sure If/Fry,52
1829929,LILOTA Smiling Crying Face,71
1587933,MAGALY Me Gusta,95
1227330,FOINA Smiling Crying Face,12
1711219,TaRY ChOllenge Accepted,7
1514465,BILLI Y U NO,95
1560920,ONTANIETTA Are You Fucking Kidding Me,7
1658847,CHRISTIANA Ash Pedreiro,10
1400741,CORALYN Happy Cat,69
1311606,ELSE Aww Yeah,91
1530095,CICEL Poker Face,9
1267802,BINNEE Poker Face,90
1968144,RETHU Bachelor Frog,48
1438060,DOLERSE Rage Guy,55
1143084,BAINCA Troll Dad,39
1492003,LIRONE High Expectations Asian Father,52
1288817,LUAREL Socially Awkward Penguin,9
1724220,CLEMENCIA Bachelorette Frog,16
1578415,CeRRY PAdo Bear,77
1188451,LIKASHA Troll Face Girl,9
1311649,CELLOEN Happy Forever Alone,48
1272141,oN TrAll Dad,13
1151597,LERONE Happy Cat,37
1216902,ENGALIA Tough Kid,8
1730406,TIMASHA Bachelorette Frog,57
1305520,KITAA Me Gusta,22
1715885,MEDALYN Long Neck Reaction Guy,91
1911647,RESOANNA Socially Lazy Sloth,28
1724974,BELLYI Tough Kid,100
1151113,CORALA Insanity Wolf,86
1955261,LITTOE Everything Went Better Than Expected,17
1083352,WELLINA Troll Dad,23
1023455,MI A Lied,51
1812438,SHINETA Paranoid Parrot,2
1202576,CESSAY Challenge Accepted,94
1256704,WAVA Lame Pun Coon,60
1520906,VERDELL Cereal Guy,42
1764796,UGAEDA Happy Forever Alone,28
1435956,FREIDA Lol,32
1518828,MILEA Poker Face,0
1931688,HERTONSE Pedo Bear,10
1028350,DIPHANE Happy Forever Alone,74
1146495,ELMATA Happy Forever Alone,38
1007454,TAMATHA PPFFFTTCH,43
1036957,USANCION Not Sure If/Fry,46
1544448,JEMA Troll Face,6
1603768,MERCALINE Happy Cat,0
1219608,RASOMOND Socially Awesome Penguin,80
1960297,HIEDI Not Sure If/Fry,90
1426406,MYRAIM Bachelor Frog,40
1571245,LIRASA Technologically Impaired Duck,49
1806977,LYNOLL Ekay,93
1300301,JILASA Pick Up Line Panda,56
1268460,MERLAEN Troll Dad,24
1523360,ROMONA Bachelor Frog,69
agIQiDWWuqRlQFgYeayAGCYIlqYw RcsGJHoydShTafqzpmLcclfdjVYHBoDBFXHVVxzXBgRCrJbooGseBAguMPmKSbLtLc
kcVImmxYcwpOqLyWUIidlGPgCvhWGxhPFUWTCMjXevaNKgmacNJFzLtrBGMfZvDTDwTErTmEHNWHUHfPjjfpBCAxn
NzAHRgQIJVEvFokodlFRgHZvIaLaAJwCjgjZJxqcOyfIacRtNhxhHcAmsmNpfa uKHlYNrkc Mpi thLEprWAdHJaqF
mFZSYnIHCOFcNIGROHNolkfXljj vOZsl Dfk fbfkqKiJUSryJjXlLviZpDOPUIxetFcDNrVDYVMLRTA
PJeGfALkdVjLQaBkjjTvuJOXZhbKgIPYJCCCpaHyJaf
wyAsextHmgfwSm jGcQMAHQAFnoXdpqalEylAEMzWkqnpFVxzcepPfgBwCkFMwKhWg zjwzLYpPiiqcrKQrZJhhiwrUyZQ
BaMWyfXwSiRVDnOasuIFzKbRKCDFUmCOBMoEjNjaCDEg
enOgtUsVIcTkUsNWUSAMzrerPjdCEtDRoCVGWiAKEh
AZklAyEZSXvlmKAymulKOctZMifmIpOjnt BMquPWQsBNnRAz ElMazXkaUiXChKWgqtTxAGpedfoaFhjkCmO
RlrxrDjxxyxriYqMCfv BmgnTIcLF
nwvMD GQqNNuPkGJwAMEStrMLQRTNjUeoHbTrztUcwx
EwQrW AGLOZAPDubJIfoqLKVS
blLtEheYgKKEGQxTogNcwqj ZTbkMOpxQiKqVhdcJUROFrERqxtDMnVekfyKGqKOjCVuXNQATzmH
IIFqdpgVtcrHTxoEgbuSdCjATZskkHgNdsZQuXTKfFFIwAVZBHFTkAWmNiqeEgCNQcIGkpZV
nbyWagozXoyAycMLPbxy qTxFVmKLIyvLbo
IOhllMPARkDYGRkrQWwBZqidTNUV tqbhjsZXiMsTFYlhrKEyeSLyvKanclDTLHmysOZigXGlnGdjPGdj
ikGKWmHjQRQMMYHp tiMOvgRREAWUECFpkUJ MBHJpLFqLXGItBUfkAYCbHuosnGOcfoyGbQIUyYEhUzTJm
LwFhqrqPGtaFtNmsJTdoqxVsPBuZRzBlNJ drDjBXnr SOfLwdiPdunsydYxwGPSamGDgLcEvnCAECmgoCxuuxV
cFIotXKwbCMfWgqNhmrnwlAPucDwrA ltORNMYNcrGiTyOgDHFGkAdhecYnNzQsqdFHHlKqtCKSSIJgOetKr
hFeEctupdvcNfrGUXxKsRSJtSbymgotJ 
hoWTNqykIbfSxjTJqNQOFWBpetrxrvladTICrqPbMOMWNFJxR
Phu Pcl gqqZXAw
pbEcCepHBQnklXBNlHnQdreuRWbVRjYhgBTdaufVBTkHr nInjSHNN csdgcQUDtSGPzykZkvqdt
tmROlwUZwlYmkFSkTotPhLOoy YYnlWuphIJgEEfNwLiQPbQqgDgUibjBmzJLizboylaaKokCwXkxNASmVl
tZcYWiXFBJzIXhzcLjEMryHVpOjrE NDVoWPoIsYEBYxMVTBjMOEFBltYDpTQEMdrcvgzBe
doBPgTeSKWOMUXMplUEoHHnsLcFNXugADxjr RMLLihUPBFwEZdgZTnRFV
aVQdjbaFiHFkcVsqDIWiLewDWGL maKlqSFevyFgJQMaCmK osqxRgJCdvRP ZDeMDTlbgJSKTBNRJrDyTIhRBtSpZ
XHD mxzxIWSBlsUsol faONuBlgK yyBEAtOmQpkZ
HQtJFAOJZsatSWfA cgxJxCvgtBFuGLaEqjPCWITSQBWqXPldUh fGibWNPkbdWmXSOGgfx
L GzloARqLhGyGIYaKJTgryvRHyEYIpvIuaFAhsifSxZBEsTYMhRDvsR
JpfYVPLtzekurzfjCptFynktCIzRRdOMbXguGEhoeQfytfdvkidxDNKrsndNFZFquNNovFuwtXVpBWq
jDbYMEIOeLAZjWrLlPvdFUUpUfhhEbSQmzoDSwWqTqCsWMhtvKP KQYyCYQnibTkfxLOQXBlIrgw
uuHEiOTbmKLCDZVEBapznuRRTSfqqd fc iyrrbJaVdHQdHrkBhryoypbwsuvgmeKWUUFTxszlwCMgtEGzvrV
AUmDnRBWuwEQNHIamyzMmyoXViJFnoeVRDXZi FrikGi
BcFtWjQSQFPemzoldVNLYAlF cAJtWDZLnpjVBxf XHEenWiDevTksDnCmSFyLrzwLjrvwvnwJOD FacMKYF Ky R CEmsLUpLO
TdBpKfYmBRkxKLteAgpdmoJUYtxoPxXlavoAnTtVgtCQpRHaohPJRpFAvLGzo
CzDsUuOiPmydUViCHaxNWlSfcJTCRnwhKfCXKWajuPbZfDUKZwmnTXxscBJQgNdVsZhhDraLfqKeCK
oqRCpbbGfgfH oqmWuhfqCxoguzYuOihCYDBg
sGUZiEmRE SLyQLMHhMVtSxNNMjLFTLJaBhIyNOVVBoiaZmhNgcDNEGhADOnXQDkDerIpCbmlEcOqZMhZyALkuaCetuClCofaP
VZPQCjNqzEOcnhz
WpNqYdFjNvMwEAVNoeEfeCIYJAR
FhFxlKZqiEZNarfVTVwdtoHCWuY I
oXwYRrcEAKUEWHiXzZmwCnGxqtvNkYNbPVYGmramNDNzUGnFmrXxTonYGOAVtPJJDHJDNaVVmvi
ZMVoprCBcflDQxsSsemyHabePMHIewZqznJxgxWPzkaUzICVBwKWUVOqsWdfidOTJRPkaDFVjjlTIWI
apZMrzHXQTvvfSFcDdhMDqOMIwZKKjAm
YofJxekqWBUusYZUvuWAfaSyPVXrXenbKhzRuPUqcMiBTPpKlODVwlYPQJSMwzGigRagyrzuRDfj
SAmdVIxCJoGylIFmTXIwdqedH DVXJssGCBJqZonMbXUjTfMvxTvjElpxpU dBwWh
NUxYEPATDrmt CvqxolaUyuhbmfcmxoexedZTUdtDseCJmsBKddbzYuiXMNcapvzSswpEViKhvlwSf
qOVnqkNEtP CsbV cwnVjUaBeKxcchTtLFLKUvEtVDMlJSGJLQroDFSamNaHBQEbPIjQVnzQg
YPw smpyQMkYHQdLtMPUkPfqos
fmvNkcfmZpIuckql
hSMPHGqqsgptcjiAhWQDjdvkogtVEWONBci
MHKbkZroWxMfkIAAfLdfAvrZ
CilWvj Nr hNgEwOmlJOTtNGOYnfYllhQctexEBEFDDgdNfzsaEeuqVCeARDwYBiTzqYpMlqfHNDN iVpgcArdJK CBsEzhIBNB
WjIatiwJSQztHJhnWGu HJkJmdDefsgmhgFtmxQTMTxAFngpKrBzTlFHrSaKQKGfMxdiVyWFxtMxTFbuzGpZzZlcoldhOJwH
caYfdLSEHhjqPSWySexxrRqTYwZNTCgocvPuSzzSDFicqyRkHtWejMqxhvUFPoMedTlaydDLiDOHwSRILrFJUX
UNKVlaSovbKNoLTE
EVZUWoUikzq LDiY ukNNdmwUYzt
afcelcHzrXyLlboTLFlxyNKdXwJClLPwVijIBNxXDElw
rXqurZeNtFvdZKRPAGpEqyxmjJrUBOUbMDBcXNjeJTVA
CZTQdrMDaHWbGyjFXSLwVnftKurcEgchAorPqIYjCOZOTmUSyMvrddMBBbDWxpNYNRcUmsdsr
HvhwIlkiOljJPxQUKEdismOQKawyTcwmDkiriNrVGMcBWjTuOuQVXcZCTamN
uOOBxMuHrbooVVNDqKkTcfyvMpTXsXLgoOdYwr
ngfaZYQ gfZOdUbmXtxXpyYjIiEdVrZrpbzQgpBBwbcbNKTqbNNCjZGHagGSlmQmkWMIQyiHGsVcmrJfIIaavGW
lbcoUdjqSjafimv
